dist/debian/control.template

```diff
@@ -13,12 +13,14 @@ Description: Scylla database main configuration file
  Scylla is a highly scalable, eventually consistent, distributed,
  partitioned row DB.
 Replaces: %{product}-server (<< 1.1)
+Replaces: %{product}-enterprise-server (<<5.5)
 Conflicts: %{product}-server (<< 1.1)
 
 Package: %{product}-server
 Architecture: any
 Depends: ${misc:Depends}, %{product}-conf (= ${binary:Version}), %{product}-python3 (= ${binary:Version})
 Replaces: %{product}-tools (<<5.5)
+Replaces: %{product}-enterprise-tools (<<5.5)
 Breaks: %{product}-tools (<<5.5)
 Description: Scylla database server binaries 
```

Comment:
What about the other packages? like scylla-enterprise, scylla-enterprise-cqlsh?

Reply:
And why 5.5?

Reply:
> What about the other packages? like scylla-enterprise, scylla-enterprise-cqlsh?

I would expect it to be there before if needed, right?